api/saturating_mul.md

```diff
@@ -0,0 +1,36 @@
+---
+api_name: saturating_mul()
+excerpt: Multiples two numbers, saturating at the numeric bounds instead of overflowing
+license: community
+toolkit: true
+topic: hyperfunctions
+tags: [hyperfunctions, saturating math]
+api_category: hyperfunction
+experimental-toolkit: true
+hyperfunction_toolkit: true
+hyperfunction_family: 'saturating math'
+hyperfunction_subfamily: saturating math
+hyperfunction_type: one-step
+---
+
+# saturating_mul()  <tag type="toolkit">Toolkit</tag><tag type="toolkit-experimental" content="Experimental" />
+The `saturating_mul` function multiples two numbers, saturating at the numeric bounds instead of overflowing.
```

Comment:
We should probably avoid the term `numeric bounds` as people might confuse this with the PostgresQL NUMERIC type.  We should probably just use (-2147483648, 2147483647) in these pages to be as descriptive as possible.

Reply:
Is it ok to still use it in the description field of the Required Arguments table or should I also remove references to it there?
Ex. "The result of `x+y`, saturating at the numeric bounds instead of overflowing. The numeric bounds are the upper and lower bounds of the 32-bit signed integers."

Reply:
Should I also remove the `numeric bounds` mention in the `Required Arguments` table?
Ex. "The result of `x-y`, saturating at the numeric bounds instead of overflowing. The numeric bounds are the upper and lower bounds of the 32-bit signed integers."